libs/net/dal/Migrations/1.0.160/Up/PostUp/01-WeekendTemplateUpdate.sql

```diff
@@ -0,0 +1,119 @@
+<div style="padding-bottom:20px;"><img src="https://dev.mmi.gov.bc.ca/assets/MMinsights_logo_black.svg" width="500px"></div>
+</div>
+<h1 id="top" style="color: #971D29; font-size: 32px; line-height: 38px; margin: 0px; border-bottom: 1px solid #6C5D62;">Evening Overview</h1>
+
+
+<div style="font-size:1.12rem; font-weight:600;"><p>@($"{Instance.PublishedOn:dddd, MMMM dd, yyyy}")</p></div>
+
+<br/>
+@if (Instance.Sections.Count() == 0)
+{
+  <p>There is no content in this report.</p>
+}
+@foreach (var section in Instance.Sections)
+{
+  var itemIndex = 0;
+  var itemCount = section.Items.Count(i => i.ItemType == AVOverviewItemType.Story);
+
+  <div class="section" style="border: 1px solid #9F9196; border-radius: 6px; margin-bottom:20px; padding: 16px;">
+
+	<div class="section-header" style="text-align:center; font-size:1.1rem; font-weight:600; padding-bottom:6px;">
+      @section.Name @(section.StartTime?.Length == 8 ? section.StartTime.Substring(0, 5) : section.StartTime) @(section.StartTime?.Length > 0 ? "hr" : "")
+      @if (!String.IsNullOrEmpty(section.Anchors)) { <span> - @section.Anchors</span> }
+    </div>
+
+
+    <table style="width:100%; border: collapse;">
+      <thead style="background-color:#9F9196;">
+        <tr>
+          <td style="text-transform:uppercase; font-weight:500; color:#FFF; padding:0 10px; width:8rem;">Placement</td>
+          <td style="text-transform:uppercase; font-weight:500; color:#FFF; padding:0 10px; width:6rem;">Time</td>
+          <td style="text-transform:uppercase; font-weight:500; color:#FFF; padding:0 10px;">Story</td>
+        </tr>
+      </thead>
+
+
+      @foreach (var item in section.Items)
+      {
+        var placement = item.ItemType switch
+        {
+          AVOverviewItemType.Intro => "Intro",
+          AVOverviewItemType.Story => itemIndex  == 1 ? "Top Story" : $"{itemIndex}/{itemCount}",
+          AVOverviewItemType.Ad => "",
+          _ => $"{item.ItemType}",
+        };
+        var time = item.ItemType switch
+        {
+          AVOverviewItemType.Ad => "",
+          _ => $"{item.Time}",
+        };
+        var summary = String.IsNullOrEmpty(item.Summary) ? "---" : item.Summary.Replace("\n", "<br/>");
+        <tr style="border-bottom: 1px solid #DED9DA;">
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@placement</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@(time?.Length == 8 ? time.Substring(0, 5) : time)</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">
+            @if (item.ContentId.HasValue)
+            {
+              <a href="@($"{ViewContentUrl}{item.ContentId}")">@summary</a>
+              @if (item.Content.FileReferences != null && item.Content.FileReferences.Any() && item.Content.FileReferences.FirstOrDefault().ContentType.StartsWith("video/") )
```

Comment:
`.FirstOrDefault()` can result in a null value.  This will fail if there is a null value.

Reply:
Added `item.Content.FileReferences.Any()` to check if there is any content and just in that case obtain the first record with contentType starting with 'video'.  If `item.Content.FileReferences.Any()` returns false then it does not proceed to the rest of the conditional if statement, so it would not execute `FirstOrDefault()`. 

Please let me know if I should check for another way to implement it.

Thank you!